### PR TITLE
the final pull request

### PR DESCRIPTION
**Pull request by samiullah12345678**
The project has the following addition 

- New and improved linters setup
- a new html and css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="styles.css">
```
Comment (samiullah12345678) [inline index.html:8]: the file attached is named styles.css and the file uploaded is a style. css

File: README.md [hunk 2]
```diff
@@ -1,5 +1 @@
-# FCC Test Suite Template
```
[the remark this answers is not in the document]
Comment (joskalenda) [reply README.md:1]: - [ ] Well, your comment is correct but not explained, why should the student agree with you on this? why do you think the readme file is not complete? this is why you need to tell the student what is missing, and why they should add it, for example, you can say that for a better repo presentation, you need to have a clear readme file and propose to the student a better way of creating it using microverse readme template.

**Comment by joskalenda** [comment]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @samiullah12345678 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Great use of github inline feature

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (samiullah12345678) [reply README.md:1]: Hey Sami the Readme file is not completed. A good project needs a good Readme to make sure any new user or reviewer can read that and establish the best possible understanding of what you want to present. Make sure you change that and happy coding!

File: index.html (hunk 1 above)
Comment (samiullah12345678) [reply index.html:8]: Hey, @samiullah12345678 you may have seen that a linter is failing. Do you know why I think that the style hint is failing you have attached the wrong file and the wrong file is uploaded by you. please fix it

**Review by NahnahAJ — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @samiullah12345678 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Good use of GitHub Inline features :heavy_check_mark: 
- Requested changes implemented :heavy_check_mark: 

## Optional suggestions
- Great job adding reasons why you request a change and making sure to sound polite and not give orders as mentioned by the previous reviewer :+1: . I will suggest that you include emojis in your reviews so it sends a friendly tone to the recipient. Also, I will suggest that you make sure your sentences are well phrased so the student understands what is expected of them and is able to deliver :100: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**